privacyidea/lib/policydecorators.py

```diff
@@ -309,6 +311,7 @@ def auth_user_passthru(wrapped_function, user_object, passw, options=None):
             if pass_thru_action in ["userstore", True]:
                 # Now we need to check the userstore password
                 if user_object.check_password(passw):
+                    g.audit_object.add_policy([p.get("name") for p in pass_thru])
```

Comment:
As ``get_policies`` returns all matching policies sorted by priority, this will add all matching passthru policies to the audit log, even though only ``pass_thru[0]`` has taken effect!

Reply:
Will think about priorities later.